tests/components/onewire/test_init.py

```diff
@@ -0,0 +1,23 @@
+"""Tests for 1-Wire config flow."""
+from homeassistant.components.onewire.const import DOMAIN
+from homeassistant.config_entries import ENTRY_STATE_LOADED, ENTRY_STATE_NOT_LOADED
+
+from . import setup_onewire_owserver_integration, setup_onewire_sysbus_integration
+
+
+async def test_unload_entry(hass):
+    """Test being able to unload an entry."""
+    config_entry_owserver = await setup_onewire_owserver_integration(hass)
+    config_entry_sysbus = await setup_onewire_sysbus_integration(hass)
+
+    assert len(hass.config_entries.async_entries(DOMAIN)) == 2
+    assert config_entry_owserver.state == ENTRY_STATE_LOADED
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)